src/Field/Instance.php

```diff
@@ -35,6 +35,7 @@ public function __construct($data) {
 
   public static function load($field_name, $entity_type, $bundle) {
     $data = \field_read_instance($entity_type, $field_name, $bundle);
+    $data = ['field' => Field::byName($field_name), 'bundle' => new Bundle($entity_type, $bundle)] + $data;
```

Comment:
Any idea why the code in `__construct` wouldn’t load the field already? I see that the bundle is probably not loaded.

Reply:
If we switch to static constructors I think we should do so consistently.